SofaKernel/modules/SofaCore/src/sofa/core/objectmodel/BaseData.h

```diff
@@ -224,7 +224,7 @@ class SOFA_CORE_API BaseData : public DDGNode
     /// @}
 
     /// If we use the Data as a link and not as value directly
-    std::string getLinkPath() const { return parentBaseData.getPath(); }
+    virtual std::string getLinkPath() const { return parentBaseData.getPath(); }
```

Comment:
This has nothing to do with the PR, but I need this to be overridable for custom Data in SofaQtQuick